test/fetch/issue-2294-patch-method.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { test } = require('tap')
```

Comment:
Why do we use `tap` rather than `node:test` here? (context: https://github.com/nodejs/undici/issues/2267)

Reply:
these tests use tap still